src/index.json

```diff
@@ -1060,6 +1060,53 @@
                     "version": "0.1.6"
                 }
             }
+        ],
+        "dms-ext": [
```

Comment:
Please rename this without the "-ext". This being an extension is implicitly assumed.